### ISSUE TITLE
[11.0] Browser console throws errors

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#8929]
### Description

After opening the newly built v11 documentation, the browser console throws errors.
Test on Safari 15.1 (17612.2.9.1.20) and Chrome  95.0.4638.69 (Official Build) (x86_64) on macOS 12.0.1 (21A559).
### Links

```
client?cae4:150 [WDS] Errors while compiling. Reload prevented.
./11.0/api/tableView.md?vue&type=template&id=0fe633fe& (./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/@vuepress/core/node_modules/.cache/vuepress","cacheIdentifier":"49b9bb6e-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/vue-loader/lib??ref--1-1!./node_modules/@vuepress/markdown-loader??ref--1-2!./.vuepress/docs-links.js!./11.0/api/tableView.md?vue&type=template&id=0fe633fe&)
Module Error (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
(Emitted value instead of an instance of Error) 

  Errors compiling template:

  tag <TBODY> has no matching end tag.

  5  |  <h3 id="tbody"><a class="header-anchor" href="#tbody">#</a> TBODY</h3>
  6  |  <a href="https://github.com/handsontable/handsontable/blob/02b383f1251b92a16acfecc11a5fa136efd15e1f/../src/tableView.js#L62" target="_blank" class="source-code-link"> Source code <OutboundLink /> </a><p><em>tableView.TBODY : HTMLTableSectionElement</em></p>
  7  |  <p>Main <TBODY> element.</p>
     |          ^^^^^^^
  8  |  <h3 id="thead"><a class="header-anchor" href="#thead">#</a> THEAD</h3>
  9  |  <a href="https://github.com/handsontable/handsontable/blob/02b383f1251b92a16acfecc11a5fa136efd15e1f/../src/tableView.js#L56" target="_blank" class="source-code-link"> Source code <OutboundLink /> </a><p><em>tableView.THEAD : HTMLTableSectionElement</em></p>

  tag <THEAD> has no matching end tag.

  8  |  <h3 id="thead"><a class="header-anchor" href="#thead">#</a> THEAD</h3>
  9  |  <a href="https://github.com/handsontable/handsontable/blob/02b383f1251b92a16acfecc11a5fa136efd15e1f/../src/tableView.js#L56" target="_blank" class="source-code-link"> Source code <OutboundLink /> </a><p><em>tableView.THEAD : HTMLTableSectionElement</em></p>
  10 |  <p>Main <THEAD> element.</p>
     |          ^^^^^^^
  11 |  <h3 id="wt"><a class="header-anchor" href="#wt">#</a> wt</h3>
  12 |  <a href="https://github.com/handsontable/handsontable/blob/02b383f1251b92a16acfecc11a5fa136efd15e1f/../src/tableView.js#L68" target="_blank" class="source-code-link"> Source code <OutboundLink /> </a><p><em>tableView.wt : <RouterLink to="/api/walkontable/">Walkontable</RouterLink></em></p>
client?cae4:159 ./11.0/api/colGroupRenderer.md?vue&type=template&id=469f7c97& (./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/@vuepress/core/node_modules/.cache/vuepress","cacheIdentifier":"49b9bb6e-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/vue-loader/lib??ref--1-1!./node_modules/@vuepress/markdown-loader??ref--1-2!./.vuepress/docs-links.js!./11.0/api/colGroupRenderer.md?vue&type=template&id=469f7c97&)
Module Error (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
(Emitted value instead of an instance of Error) 

  Errors compiling template:

  tag <colgroup> has no matching end tag.

  4  |  <h2 id="description"><a class="header-anchor" href="#description">#</a> Description</h2>
  5  |  <p>Colgroup renderer responsible for managing (inserting, tracking, rendering) COL elements.</p>
  6  |    <colgroup> (root node)
     |    ^^^^^^^^^^
  7  |      ├ <col>   \
  8  |      ├ <col>    \
./11.0/api/cellsRenderer.md?vue&type=template&id=122d0a86& (./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/@vuepress/core/node_modules/.cache/vuepress","cacheIdentifier":"49b9bb6e-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/vue-loader/lib??ref--1-1!./node_modules/@vuepress/markdown-loader??ref--1-2!./.vuepress/docs-links.js!./11.0/api/cellsRenderer.md?vue&type=template&id=122d0a86&)
Module Error (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
(Emitted value instead of an instance of Error) 

  Errors compiling template:

  tag <td> has no matching end tag.

  38 |      ├ <td>     - CellsRenderer
  39 |      ├ <td>    /
  40 |      └ <td>   /.
     |        ^^^^
  41 |  </ContentSlotsDistributor>
  42 |  

  tag <th> has no matching end tag.

  33 |  <p>Row headers renderer responsible for managing (inserting, tracking, rendering) TR elements belongs to TR.</p>
  34 |    <tr> (root node)
  35 |      ├ <th>   --- RowHeadersRenderer
     |        ^^^^
  36 |      ├ <td>   \
  37 |      ├ <td>    \

  tag <tr> has no matching end tag.

  32 |  <h2 id="description-4"><a class="header-anchor" href="#description-4">#</a> Description</h2>
  33 |  <p>Row headers renderer responsible for managing (inserting, tracking, rendering) TR elements belongs to TR.</p>
  34 |    <tr> (root node)
     |    ^^^^
  35 |      ├ <th>   --- RowHeadersRenderer
  36 |      ├ <td>   \

  tag <td> has no matching end tag.

  29 |      ├ <td>     - CellsRenderer
  30 |      ├ <td>    /
  31 |      └ <td>   /.
     |        ^^^^
  32 |  <h2 id="description-4"><a class="header-anchor" href="#description-4">#</a> Description</h2>
  33 |  <p>Row headers renderer responsible for managing (inserting, tracking, rendering) TR elements belongs to TR.</p>

  tag <th> has no matching end tag.

  24 |  <p>Cell renderer responsible for managing (inserting, tracking, rendering) TD elements.</p>
  25 |    <tr> (root node)
  26 |      ├ <th>   --- RowHeadersRenderer
     |        ^^^^
  27 |      ├ <td>   \
  28 |      ├ <td>    \

  tag <tr> has no matching end tag.

  23 |  <h2 id="description-3"><a class="header-anchor" href="#description-3">#</a> Description</h2>
  24 |  <p>Cell renderer responsible for managing (inserting, tracking, rendering) TD elements.</p>
  25 |    <tr> (root node)
     |    ^^^^
  26 |      ├ <th>   --- RowHeadersRenderer
  27 |      ├ <td>   \

  tag <td> has no matching end tag.

  19 |      ├ <td>     - CellsRenderer
  20 |      ├ <td>    /
  21 |      └ <td>   /.
     |        ^^^^
  22 |  <h2 id="methods"><a class="header-anchor" href="#methods">#</a> Methods</h2>
  23 |  <h2 id="description-3"><a class="header-anchor" href="#description-3">#</a> Description</h2>

  tag <th> has no matching end tag.

  14 |  <p>Row headers renderer responsible for managing (inserting, tracking, rendering) TR elements belongs to TR.</p>
  15 |    <tr> (root node)
  16 |      ├ <th>   --- RowHeadersRenderer
     |        ^^^^
  17 |      ├ <td>   \
  18 |      ├ <td>    \

  tag <tr> has no matching end tag.

  13 |  <h2 id="description-2"><a class="header-anchor" href="#description-2">#</a> Description</h2>
  14 |  <p>Row headers renderer responsible for managing (inserting, tracking, rendering) TR elements belongs to TR.</p>
  15 |    <tr> (root node)
     |    ^^^^
  16 |      ├ <th>   --- RowHeadersRenderer
  17 |      ├ <td>   \

  tag <td> has no matching end tag.

  10 |      ├ <td>     - CellsRenderer
  11 |      ├ <td>    /
  12 |      └ <td>   /.
     |        ^^^^
  13 |  <h2 id="description-2"><a class="header-anchor" href="#description-2">#</a> Description</h2>
  14 |  <p>Row headers renderer responsible for managing (inserting, tracking, rendering) TR elements belongs to TR.</p>

  tag <th> has no matching end tag.

  5  |  <p>Cell renderer responsible for managing (inserting, tracking, rendering) TD elements.</p>
  6  |    <tr> (root node)
  7  |      ├ <th>   --- RowHeadersRenderer
     |        ^^^^
  8  |      ├ <td>   \
  9  |      ├ <td>    \

  tag <tr> has no matching end tag.

  4  |  <h2 id="description"><a class="header-anchor" href="#description">#</a> Description</h2>
  5  |  <p>Cell renderer responsible for managing (inserting, tracking, rendering) TD elements.</p>
  6  |    <tr> (root node)
     |    ^^^^
  7  |      ├ <th>   --- RowHeadersRenderer
  8  |      ├ <td>   \

errors @ client?cae4:159
eval @ socket.js?e29c:47
sock.onmessage @ SockJSClient.js?0a33:67
EventTarget.dispatchEvent @ sockjs.js?9be2:170
eval @ sockjs.js?9be2:891
SockJS._transportMessage @ sockjs.js?9be2:889
EventEmitter.emit @ sockjs.js?9be2:86
WebSocketTransport.ws.onmessage @ sockjs.js?9be2:2965
./11.0/api/columnHeadersRenderer.md?vue&type=template&id=6024255e& (./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/@vuepress/core/node_modules/.cache/vuepress","cacheIdentifier":"49b9bb6e-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/vue-loader/lib??ref--1-1!./node_modules/@vuepress/markdown-loader??ref--1-2!./.vuepress/docs-links.js!./11.0/api/columnHeadersRenderer.md?vue&type=template&id=6024255e&)
Module Error (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
(Emitted value instead of an instance of Error) 

  Errors compiling template:

  tag <tr> has no matching end tag.

  9  |      ├ <tr>     - ColumnHeadersRenderer
  10 |      ├ <tr>    /
  11 |      └ <tr>   /.
     |        ^^^^
  12 |  </ContentSlotsDistributor>
  13 |  

  tag <thead> has no matching end tag.

  4  |  <h2 id="description"><a class="header-anchor" href="#description">#</a> Description</h2>
  5  |  <p>Column headers renderer responsible for managing (inserting, tracking, rendering) TR and TH elements.</p>
  6  |    <thead> (root node)
     |    ^^^^^^^
  7  |      ├ <tr>   \
  8  |      ├ <tr>    \

errors @ client?cae4:159
eval @ socket.js?e29c:47
sock.onmessage @ SockJSClient.js?0a33:67
EventTarget.dispatchEvent @ sockjs.js?9be2:170
eval @ sockjs.js?9be2:891
SockJS._transportMessage @ sockjs.js?9be2:889
EventEmitter.emit @ sockjs.js?9be2:86
WebSocketTransport.ws.onmessage @ sockjs.js?9be2:2965
```

**Comment by aninde** [comment]
This issue is fixed on the current `release/11.0.0` branch - today's build.

**Comment by AMBudnik** [comment]
No errors are shown in the latest version of the documentation. I'm adding the `Released` label.
